### PR TITLE
Add `len` method for `MPCTensor`

### LINKED ISSUE #182: No len for MPCTensor
## Feature Description
There is no `len` method for `MPCTensor` now, but this have in `torch.tensor`.
this is  method is very useful for new beginner.

### PR DESCRIPTION
**Pull request by Syzygianinfern0**
## Description
Closes #182

I think that should be it :stuck_out_tongue:

## Affected Dependencies
None.

## How has this been tested?
- Added a test

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by aanurraj — APPROVED** [review]
🎉 🤣

**Comment by kamathhrishi** [comment]
Looks good to me.

**Review by kamathhrishi — APPROVED** [review]
